app/controllers/api/v0/users_controller.rb

```diff
@@ -62,9 +62,13 @@ def token
 
   private
 
-    def show_user(user, show_rankings: false)
```

Comment:
The name `current_user` for this flag is definitely confusing. In the context of this controller, this name is already defined to retrieve the actual `User` instance (or `nil` if you're not logged in). So you definitely have to pick another name.